[main] Upgrade to latest dependencies

knative-automation opened the pull request:
Cron -knative-prow-robot

/cc knative-sandbox/client-writers
/assign $knative-sandbox/client-writers

Produced by: knative-sandbox/knobots/actions/update-deps

Details:
```
bumping knative.dev/caching 5691bb3...3d48f8d:
  > 3d48f8d Update actions (# 464)
  > 1fab472 upgrade to latest dependencies (# 463)
  > bbc9e06 upgrade to latest dependencies (# 462)
  > dfd51be upgrade to latest dependencies (# 461)
  > 0643a27 upgrade to latest dependencies (# 460)
  > 9db8644 upgrade to latest dependencies (# 459)
  > 088b702 upgrade to latest dependencies (# 458)
  > c2674e6 Add markready method for image (# 457)
  > 509f22e Update knative/community files (# 455)
  > 9f992d1 upgrade to latest dependencies (# 456)
  > a653515 change status to duckv1.status, add reconciler framework  (# 454)
  > bfe6379 upgrade to latest dependencies (# 452)
  > 08a0136 upgrade to latest dependencies (# 451)
  > 1f173ac Update knative/community files (# 450)
  > ec01baa upgrade to latest dependencies (# 449)
  > aa595e3 Integrate the gopath hacks to allow for codegen anywhere. (# 398)
  > bcbf2ed Update knative/community files (# 447)
  > 53a85b1 upgrade to latest dependencies (# 448)
  > a928dbe Update OWNERS_ALIASES to match autogen in community (# 446)
bumping knative.dev/hack b6ab329...93ad912:
  > 93ad912 Print java and mvn version (# 65)
  > 86f9adc Fix default codegen pkg dir (# 67)
  > 88c69cd Add debug info for release.sh to investigate https://knative.slack.com/archives/CA1DTGZ2N/p1619160369065000 (# 66)
  > b5f6e25 Update OWNERS_ALIASES to match autogen in community (# 62)
bumping knative.dev/serving 813aa65...0cc2c38:
  > 0cc2c38 upgrade to latest dependencies (# 11324)
  > bcc1eef Update net-contour nightly (# 11305)
  > 8062d86 Use test options for feature tests and ha tests (# 11319)
  > 9997277 Remove responsive-revision-gc config from config-features (# 11309)
  > 4272029 Disable webhook for net-certmanager-webhook (# 11315)
  > 571000c Update net-kourier nightly (# 11312)
  > 99acbaf Do not call GetIngressEndpoint() when --resolvabledomain is enabled (# 11318)
  > d30f0bd Update net-certmanager nightly (# 11306)
  > c88dd37 Update net-istio nightly (# 11311)
  > f8f26b7 Do not toggle configuration in e2e when unnecessary (# 11259)
  > d436414 Update actions (# 11304)
  > 568f0ca ensure our temp files has a yaml extension - otherwise ytt will ignore it (# 11303)
  > 2d4e4a1 Make digest resolution idempotent (# 11293)
  > dd0e497 Update go-containerregistry to v0.5.0 (# 11292)
  > dd4e1cb Increase digest resolution rate limit (# 11279)
  > 91d511c Update net-kourier nightly (# 11301)
  > 28e53cb Update net-contour nightly (# 11302)
  > 69214d4 Update net-istio nightly (# 11300)
  > ad94366 Change magic DNS to sslip.io (# 11298)
  > c2f3fda Make sure the decider state is settled in test (# 11290)
  > 695b9e7 Update net-contour nightly (# 11288)
  > aa2ef62 Update net-istio nightly (# 11287)
  > f9abb7f Update net-certmanager nightly (# 11282)
  > e2a8237 e2e - drop imperative bash yaml for declarative ytt/kapp (# 11175)
  > e04223b remove duplicate additionalPrinterColumns (# 11285)
  > 08d17a2 Make KO_FLAG flexible for e2e-tests (# 11284)
  > 5ad9c8e Implement per-item rate limits for digest resolution (# 11278)
  > 101c6f1 Add complete schemas to all CRDs (# 11244)
  > 6db455b Update net-contour nightly (# 11267)
  > 7a9e98b Update net-kourier nightly (# 11281)
  > 9a1d53f Update net-istio nightly (# 11280)
  > 1c96334 upgrade to latest dependencies (# 11275)
  > 3bb256d Update net-kourier nightly (# 11274)
  > f712820 Drop ingressClass and certificateClass flags from serving test (# 11277)
  > f60aa3b Use non-pointer workItem in digest resolution (# 11273)
  > 5af4e84 Make sure deployments never underscale on rollouts (# 11140)
  > 1cdf79f Only print deployment state in error cases (# 11272)
  > 6ed3b03 Update net-kourier nightly (# 11265)
  > 38c791e upgrade to latest dependencies (# 11271)
  > c28723a upgrade to latest dependencies (# 11269)
  > efb7b45 Update net-istio nightly (# 11268)
  > 1297dbd Update net-certmanager nightly (# 11266)
  > da0d742 Actually use the rate limited queue (# 11256)
  > 5692160 Update visibility label comment in config-domain (# 11253)
  > 88ff33f Update net-istio nightly (# 11247)
  > b69efda Update net-kourier nightly (# 11255)
  > 7ce1ba1 Update net-contour nightly (# 11254)
  > b4dd584 upgrade to latest dependencies (# 11264)
  > c7326c1 Add myself to support rotation of 04-26 (# 11263)
  > 28ba542 upgrade to latest dependencies (# 11262)
  > 792c8d2 upgrade to latest dependencies (# 11260)
  > 3537984 Don't fall back to cluster IP when error is not mesh-related (# 11208)
  > e97bc76 upgrade to latest dependencies (# 11252)
  > d31ab23 Lower the drain sleep in queue-proxy to previous value (# 11249)
  > 3214502 upgrade to latest dependencies (# 11246)
  > 4ac36ce Add API doc generation to codegen (# 11238)
  > a370537 hardcode the ambassador namespace (# 11245)
  > 3844721 don't specify patch and build for gke version (# 11214)
  > 0470900 Update net-kourier nightly (# 11242)
  > 82e650a Update net-istio nightly (# 11241)
  > 152af6d Update net-contour nightly (# 11240)
  > 6ebf113 Use OnDeletionInterface to drop potentially racy direct usages of DeleteFunc (# 11235)
  > 20e7d89 Update net-kourier nightly (# 11233)
  > 258834f Update net-contour nightly (# 11232)
  > ef2ac45 Use new helper to detect potential mesh errors (# 11234)
  > 5b67bd9 don't use vendor in ko:// path (# 11229)
  > f43af48 Update knative/community files (# 11227)
  > 3822eac upgrade to latest dependencies (# 11228)
  > b2fdf93 drop unused e2e test flag (# 11230)
  > e4c7848 Update net-kourier nightly (# 11219)
  > 83808bc Update net-contour nightly (# 11220)
  > d2ca2eb Drop unnecessary parsing from/to string in probe (# 11224)
  > ae54012 Update net-istio nightly (# 11221)
  > 912936a Allow for more overscale in tests if the deployment is being rolled (# 11209)
  > 1b83b4b upgrade to latest dependencies (# 11217)
  > 9a5f4b8 Update net-istio nightly (# 11180)
  > 44f6033 Fix activator index update log (# 11203)
  > 40a54e3 Statically configure passthrough lb in the activator (# 11184)
  > 9054d8b Don't hardcode replica count in the Activator HA test (# 11189)
  > b6af5b6 Update net-contour nightly (# 11182)
  > ef78b9a Replace all copies of autoscaling annotations with constants (# 11213)
  > db9538d Add addressable-resolver role for serving controller (# 10693)
  > bdc0db4 use GKE 1.18 for CI until we sort out 1.19 failures (# 11206)
  > bbfa917 Set useragent for digest resolver (# 10590)
  > 935bcc8 Excluse domainmapping-webhook from chaos (# 11202)
  > 0640781 Add specific port to instruct istio to open correct listener (# 11063)
  > c50bbf8 Update net-kourier nightly (# 11191)
  > 3b198ec Fix couple of nits in scale tests (# 11204)
  > 09f07d7 bump kind e2e to use go1.15 (# 11187)
  > 0751aed Update net-kourier nightly (# 11181)
  > 7f889d8 fix broken link in spec error.md (# 11167)
  > 3792fb4 Add test for existing queue probe behaviour (# 11186)
  > e57b86b Revert "Thread network config into the autoscaler (# 11056)" (# 11185)
  > 53df47c Remove private service listing (# 11178)
  > 9faf9e6 Don't require ActivatorOverloadTest to scale to 0 (# 11177)
  > 942af3f Move comment above referenced code (# 11179)
  > 40e7ad0 Update net-kourier nightly (# 11169)
  > bd8d6a0 Use passthrough loadbalancing in activator state keeping (# 11172)
  > 7ac0c4f Add a private service name helper and use throughout (# 11176)
  > fc6ca85 Subdomain url from traffic status (# 11149)
  > 01ef03f Be more discerning about which errors cause mesh fall-back (# 11174)
  > 8a0eff7 Connect the networking config with stats scraping (# 11161)
  > d73eb99 upgrade to latest dependencies (# 11173)
  > 2bfe3de upgrade to latest dependencies (# 11168)
  > c96e749 fix cluster.local replace directive (# 11166)
  > a4f70f6 Cleanup deepcopygen annotations and calls (# 11165)
  > 3f6d931 Override Host in activator's main handler if option is set (# 11162)
  > 8d72e76 Fix a few wrong logs in tests and make them consistent (# 11164)
  > 6ed9b72 upgrade to latest dependencies (# 11163)
  > 9cabf66 Update net-contour nightly (# 11157)
  > 05ae6b2 Update net-kourier nightly (# 11158)
  > 81e36a1 Update net-istio nightly (# 11156)
  > 1b007c5 upgrade to latest dependencies (# 11155)
  > daab3e3 Update net-kourier nightly (# 11144)
  > b2c6829 Update net-certmanager nightly (# 11145)
  > b2f616b Update net-contour nightly (# 11143)
  > 339d7e2 Thread network config into the activator (# 11052)
  > 443038c Update knative/community files (# 11141)
  > cc3f45b Update net-contour nightly (# 11131)
  > 6396164 Update OWNERS_ALIASES to match autogen in community (# 11103)
  > d59d81f upgrade to latest dependencies (# 11138)
  > 47f56c6 run kind e2e in release branches (# 11127)
  > 79d3123 Drop extra revision check from MinScale test (# 11130)
  > 3cc16ca Remove JRBANCEL from support rotation. (# 11129)
  > f4edec6 Use similar rollout check to kubectl in autoscale test (# 11123)
  > 983a307 Revision return replicas (# 11038)
  > 1f297ad expand (# 11119)
bumping knative.dev/networking 999a770...a3028d5:
  > a3028d5 Update actions (# 405)
  > 1ad047c upgrade to latest dependencies (# 404)
  > 796a800 upgrade to latest dependencies (# 403)
  > b62df94 upgrade to latest dependencies (# 402)
  > 2696fca Add missing JSON tag to protocol type (# 400)
  > 7a7ad51 upgrade to latest dependencies (# 401)
  > 2ee665e upgrade to latest dependencies (# 398)
  > 2b84569 upgrade to latest dependencies (# 397)
  > 57bee66 Allow the prober to cancel probing by key (# 396)
  > 12abdf2 Add helper to test for potential mesh errors (# 393)
  > de8ac12 upgrade to latest dependencies (# 395)
  > 4e9d153 Update knative/community files (# 394)
  > 90f951c upgrade to latest dependencies (# 392)
  > a64b4be upgrade to latest dependencies (# 391)
  > 89774aa Append headers to TestRule to verify we hit expected backend (# 385)
  > e294898 upgrade to latest dependencies (# 390)
  > 945b81a Update knative/community files (# 389)
  > c8c1ee6 upgrade to latest dependencies (# 387)
  > fe8b23b Clean up DeprecatedDefaultIngressClassKey (# 388)
  > e38eb2b Update knative/community files (# 386)
  > b02df84 Update OWNERS_ALIASES to match autogen in community (# 383)
  > 89f074b Add main branch in knative-downstream.yaml to trigger pull rquest (# 384)
  > 63985f0 Update knative/community files (# 379)
```

maximilien reviewed (dismissed):
/approve
/lgtm
/hold until next release so this PR is updated instead of new one

maximilien commented:
/unhold

cdlliuy commented:
/lgtm
/approve

knative-automation committed:
upgrade to latest dependencies

bumping knative.dev/caching 5691bb3...3d48f8d:
  > 3d48f8d Update actions (# 464)
  > 1fab472 upgrade to latest dependencies (# 463)
  > bbc9e06 upgrade to latest dependencies (# 462)
  > dfd51be upgrade to latest dependencies (# 461)
  > 0643a27 upgrade to latest dependencies (# 460)
  > 9db8644 upgrade to latest dependencies (# 459)
  > 088b702 upgrade to latest dependencies (# 458)
  > c2674e6 Add markready method for image (# 457)
  > 509f22e Update knative/community files (# 455)
  > 9f992d1 upgrade to latest dependencies (# 456)
  > a653515 change status to duckv1.status, add reconciler framework  (# 454)
  > bfe6379 upgrade to latest dependencies (# 452)
  > 08a0136 upgrade to latest dependencies (# 451)
  > 1f173ac Update knative/community files (# 450)
  > ec01baa upgrade to latest dependencies (# 449)
  > aa595e3 Integrate the gopath hacks to allow for codegen anywhere. (# 398)
  > bcbf2ed Update knative/community files (# 447)
  > 53a85b1 upgrade to latest dependencies (# 448)
  > a928dbe Update OWNERS_ALIASES to match autogen in community (# 446)
bumping knative.dev/hack b6ab329...93ad912:
  > 93ad912 Print java and mvn version (# 65)
  > 86f9adc Fix default codegen pkg dir (# 67)
  > 88c69cd Add debug info for release.sh to investigate https://knative.slack.com/archives/CA1DTGZ2N/p1619160369065000 (# 66)
  > b5f6e25 Update OWNERS_ALIASES to match autogen in community (# 62)
bumping knative.dev/serving 813aa65...0cc2c38:
  > 0cc2c38 upgrade to latest dependencies (# 11324)
  > bcc1eef Update net-contour nightly (# 11305)
  > 8062d86 Use test options for feature tests and ha tests (# 11319)
  > 9997277 Remove responsive-revision-gc config from config-features (# 11309)
  > 4272029 Disable webhook for net-certmanager-webhook (# 11315)
  > 571000c Update net-kourier nightly (# 11312)
  > 99acbaf Do not call GetIngressEndpoint() when --resolvabledomain is enabled (# 11318)
  > d30f0bd Update net-certmanager nightly (# 11306)
  > c88dd37 Update net-istio nightly (# 11311)
  > f8f26b7 Do not toggle configuration in e2e when unnecessary (# 11259)
  > d436414 Update actions (# 11304)
  > 568f0ca ensure our temp files has a yaml extension - otherwise ytt will ignore it (# 11303)
  > 2d4e4a1 Make digest resolution idempotent (# 11293)
  > dd0e497 Update go-containerregistry to v0.5.0 (# 11292)
  > dd4e1cb Increase digest resolution rate limit (# 11279)
  > 91d511c Update net-kourier nightly (# 11301)
  > 28e53cb Update net-contour nightly (# 11302)
  > 69214d4 Update net-istio nightly (# 11300)
  > ad94366 Change magic DNS to sslip.io (# 11298)
  > c2f3fda Make sure the decider state is settled in test (# 11290)
  > 695b9e7 Update net-contour nightly (# 11288)
  > aa2ef62 Update net-istio nightly (# 11287)
  > f9abb7f Update net-certmanager nightly (# 11282)
  > e2a8237 e2e - drop imperative bash yaml for declarative ytt/kapp (# 11175)
  > e04223b remove duplicate additionalPrinterColumns (# 11285)
  > 08d17a2 Make KO_FLAG flexible for e2e-tests (# 11284)
  > 5ad9c8e Implement per-item rate limits for digest resolution (# 11278)
  > 101c6f1 Add complete schemas to all CRDs (# 11244)
  > 6db455b Update net-contour nightly (# 11267)
  > 7a9e98b Update net-kourier nightly (# 11281)
  > 9a1d53f Update net-istio nightly (# 11280)
  > 1c96334 upgrade to latest dependencies (# 11275)
  > 3bb256d Update net-kourier nightly (# 11274)
  > f712820 Drop ingressClass and certificateClass flags from serving test (# 11277)
  > f60aa3b Use non-pointer workItem in digest resolution (# 11273)
  > 5af4e84 Make sure deployments never underscale on rollouts (# 11140)
  > 1cdf79f Only print deployment state in error cases (# 11272)
  > 6ed3b03 Update net-kourier nightly (# 11265)
  > 38c791e upgrade to latest dependencies (# 11271)
  > c28723a upgrade to latest dependencies (# 11269)
  > efb7b45 Update net-istio nightly (# 11268)
  > 1297dbd Update net-certmanager nightly (# 11266)
  > da0d742 Actually use the rate limited queue (# 11256)
  > 5692160 Update visibility label comment in config-domain (# 11253)
  > 88ff33f Update net-istio nightly (# 11247)
  > b69efda Update net-kourier nightly (# 11255)
  > 7ce1ba1 Update net-contour nightly (# 11254)
  > b4dd584 upgrade to latest dependencies (# 11264)
  > c7326c1 Add myself to support rotation of 04-26 (# 11263)
  > 28ba542 upgrade to latest dependencies (# 11262)
  > 792c8d2 upgrade to latest dependencies (# 11260)
  > 3537984 Don't fall back to cluster IP when error is not mesh-related (# 11208)
  > e97bc76 upgrade to latest dependencies (# 11252)
  > d31ab23 Lower the drain sleep in queue-proxy to previous value (# 11249)
  > 3214502 upgrade to latest dependencies (# 11246)
  > 4ac36ce Add API doc generation to codegen (# 11238)
  > a370537 hardcode the ambassador namespace (# 11245)
  > 3844721 don't specify patch and build for gke version (# 11214)
  > 0470900 Update net-kourier nightly (# 11242)
  > 82e650a Update net-istio nightly (# 11241)
  > 152af6d Update net-contour nightly (# 11240)
  > 6ebf113 Use OnDeletionInterface to drop potentially racy direct usages of DeleteFunc (# 11235)
  > 20e7d89 Update net-kourier nightly (# 11233)
  > 258834f Update net-contour nightly (# 11232)
  > ef2ac45 Use new helper to detect potential mesh errors (# 11234)
  > 5b67bd9 don't use vendor in ko:// path (# 11229)
  > f43af48 Update knative/community files (# 11227)
  > 3822eac upgrade to latest dependencies (# 11228)
  > b2fdf93 drop unused e2e test flag (# 11230)
  > e4c7848 Update net-kourier nightly (# 11219)
  > 83808bc Update net-contour nightly (# 11220)
  > d2ca2eb Drop unnecessary parsing from/to string in probe (# 11224)
  > ae54012 Update net-istio nightly (# 11221)
  > 912936a Allow for more overscale in tests if the deployment is being rolled (# 11209)
  > 1b83b4b upgrade to latest dependencies (# 11217)
  > 9a5f4b8 Update net-istio nightly (# 11180)
  > 44f6033 Fix activator index update log (# 11203)
  > 40a54e3 Statically configure passthrough lb in the activator (# 11184)
  > 9054d8b Don't hardcode replica count in the Activator HA test (# 11189)
  > b6af5b6 Update net-contour nightly (# 11182)
  > ef78b9a Replace all copies of autoscaling annotations with constants (# 11213)
  > db9538d Add addressable-resolver role for serving controller (# 10693)
  > bdc0db4 use GKE 1.18 for CI until we sort out 1.19 failures (# 11206)
  > bbfa917 Set useragent for digest resolver (# 10590)
  > 935bcc8 Excluse domainmapping-webhook from chaos (# 11202)
  > 0640781 Add specific port to instruct istio to open correct listener (# 11063)
  > c50bbf8 Update net-kourier nightly (# 11191)
  > 3b198ec Fix couple of nits in scale tests (# 11204)
  > 09f07d7 bump kind e2e to use go1.15 (# 11187)
  > 0751aed Update net-kourier nightly (# 11181)
  > 7f889d8 fix broken link in spec error.md (# 11167)
  > 3792fb4 Add test for existing queue probe behaviour (# 11186)
  > e57b86b Revert "Thread network config into the autoscaler (# 11056)" (# 11185)
  > 53df47c Remove private service listing (# 11178)
  > 9faf9e6 Don't require ActivatorOverloadTest to scale to 0 (# 11177)
  > 942af3f Move comment above referenced code (# 11179)
  > 40e7ad0 Update net-kourier nightly (# 11169)
  > bd8d6a0 Use passthrough loadbalancing in activator state keeping (# 11172)
  > 7ac0c4f Add a private service name helper and use throughout (# 11176)
  > fc6ca85 Subdomain url from traffic status (# 11149)
  > 01ef03f Be more discerning about which errors cause mesh fall-back (# 11174)
  > 8a0eff7 Connect the networking config with stats scraping (# 11161)
  > d73eb99 upgrade to latest dependencies (# 11173)
  > 2bfe3de upgrade to latest dependencies (# 11168)
  > c96e749 fix cluster.local replace directive (# 11166)
  > a4f70f6 Cleanup deepcopygen annotations and calls (# 11165)
  > 3f6d931 Override Host in activator's main handler if option is set (# 11162)
  > 8d72e76 Fix a few wrong logs in tests and make them consistent (# 11164)
  > 6ed9b72 upgrade to latest dependencies (# 11163)
  > 9cabf66 Update net-contour nightly (# 11157)
  > 05ae6b2 Update net-kourier nightly (# 11158)
  > 81e36a1 Update net-istio nightly (# 11156)
  > 1b007c5 upgrade to latest dependencies (# 11155)
  > daab3e3 Update net-kourier nightly (# 11144)
  > b2c6829 Update net-certmanager nightly (# 11145)
  > b2f616b Update net-contour nightly (# 11143)
  > 339d7e2 Thread network config into the activator (# 11052)
  > 443038c Update knative/community files (# 11141)
  > cc3f45b Update net-contour nightly (# 11131)
  > 6396164 Update OWNERS_ALIASES to match autogen in community (# 11103)
  > d59d81f upgrade to latest dependencies (# 11138)
  > 47f56c6 run kind e2e in release branches (# 11127)
  > 79d3123 Drop extra revision check from MinScale test (# 11130)
  > 3cc16ca Remove JRBANCEL from support rotation. (# 11129)
  > f4edec6 Use similar rollout check to kubectl in autoscale test (# 11123)
  > 983a307 Revision return replicas (# 11038)
  > 1f297ad expand (# 11119)
bumping knative.dev/networking 999a770...a3028d5:
  > a3028d5 Update actions (# 405)
  > 1ad047c upgrade to latest dependencies (# 404)
  > 796a800 upgrade to latest dependencies (# 403)
  > b62df94 upgrade to latest dependencies (# 402)
  > 2696fca Add missing JSON tag to protocol type (# 400)
  > 7a7ad51 upgrade to latest dependencies (# 401)
  > 2ee665e upgrade to latest dependencies (# 398)
  > 2b84569 upgrade to latest dependencies (# 397)
  > 57bee66 Allow the prober to cancel probing by key (# 396)
  > 12abdf2 Add helper to test for potential mesh errors (# 393)
  > de8ac12 upgrade to latest dependencies (# 395)
  > 4e9d153 Update knative/community files (# 394)
  > 90f951c upgrade to latest dependencies (# 392)
  > a64b4be upgrade to latest dependencies (# 391)
  > 89774aa Append headers to TestRule to verify we hit expected backend (# 385)
  > e294898 upgrade to latest dependencies (# 390)
  > 945b81a Update knative/community files (# 389)
  > c8c1ee6 upgrade to latest dependencies (# 387)
  > fe8b23b Clean up DeprecatedDefaultIngressClassKey (# 388)
  > e38eb2b Update knative/community files (# 386)
  > b02df84 Update OWNERS_ALIASES to match autogen in community (# 383)
  > 89f074b Add main branch in knative-downstream.yaml to trigger pull rquest (# 384)
  > 63985f0 Update knative/community files (# 379)

Signed-off-by: Knative Automation <automation@knative.team>

maximilien reviewed (approved):
/ok-to-test
/approve
/lgtm
